src/app/components/preview-card/preview-card.component.scss

```diff
@@ -0,0 +1,60 @@
+.mat-card {
+  position: relative;
+  margin: 20px 10px;
+  transition: 0.2s;
+  transition-property: box-shadow;
+
+  &:hover, &:focus {
+    box-shadow: 0 5px 11px 0 rgba(0, 0, 0, 0.18),
+      0 4px 15px 0 rgba(0, 0, 0, 0.15);
+  }
+}
+
+.mat-card-title {
+  margin-left: -16px; // align header tile with content (mat-card-text not responsible)
+}
+
+.tags-box {
```

Comment:
If there are many tags or if a tag is really long, the box could potentially cover the platform icon, e.g.

<img width="681" alt="Screen Shot 2021-06-30 at 7 30 55 PM" src="https://user-images.githubusercontent.com/9377970/124056003-c91a3280-d9d9-11eb-82c6-2cec8698a91c.png">

Reply:
Good point! I never tested for super long tag. The longest tagId we have now is 24 chars, but the tag should not be long than the 60. ([tagId schema](https://github.com/Sage-Bionetworks/rocc-schemas/blob/main/openapi/components/parameters/TagId.yaml)). We could either set short length of tagId in schema or make responsive to long tagIds with vertically layout.

Reply:
@rrchai to open a new ticket describing this issue before including the link here and resolving the conversation.